Sources/AccountSDKIOSWeb/Lib/AuthState.swift

```diff
@@ -24,9 +24,9 @@ internal struct AuthState: Codable {
 
 extension AuthState {
 
-    init(mfa: MFAType?) {
+    init(mfa: MFAType?, state: String?) {
 
-        let state = randomString(length: 10)
+        let state = state ?? randomString(length: 10)
```

Comment:
I would leave some comment here explaining why this logic appears (meaning: we want to give access to add `state`, yet have to keep it non-empty as `Identity` object might expect it).

Reply:
Sure, will add it 👍